specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.go.md

```diff
@@ -0,0 +1,45 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go) && !$(track2)
+go:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  clear-output-folder: true
+```
+
+``` yaml $(go) && $(track2)
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/containerservice/armcontainerservicefleet
```

Comment:
New `readme.go` for fleet, modified `module-name` from `sdk/resourcemanager/containerservice/armcontainerservice` to `sdk/resourcemanager/containerservice/armcontainerservicefleet`.